src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIRangeGuard.java

```diff
@@ -178,24 +183,38 @@ protected AIState searchTarget()
      */
     protected AIState huntDown()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 7, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)